src/pylutron_caseta/smartbridge.py

```diff
@@ -57,10 +57,12 @@ def __init__(
         self.scenes: Dict[str, dict] = {}
         self.occupancy_groups: Dict[str, dict] = {}
         self.areas: Dict[str, dict] = {}
+        self.smart_away_state: str = ""
         self._connect = connect
         self._subscribers: Dict[str, Callable[[], None]] = {}
         self._occupancy_subscribers: Dict[str, Callable[[], None]] = {}
         self._button_subscribers: Dict[str, Callable[[str], None]] = {}
+        self._smart_away_subscribers: Callable[[str], None] = {}
```

Comment:
```suggestion
        self._smart_away_subscriber: Callable[[str], None] = {}
```

If there's only one it doesn't need to be plural.